Wabbajack.Common/Paths.cs

```diff
@@ -264,7 +266,7 @@ public AbsolutePath AppendToName(string toAppend)
 
         public AbsolutePath Combine(params RelativePath[] paths)
         {
-            return new AbsolutePath(Path.Combine(paths.Select(s => (string)s).Where(s => s != null).Cons(_path).ToArray()));
+            return new AbsolutePath(Path.Combine(paths.Select(s => (string)s).Cons(_path).ToArray()));
```

Comment:
**Example of speed upside to adding nullability**
Since the variable contains types that can never be null, we can omit the check for nulls from our logic.